test/template-tests/testcases/deploymentTemplate/Virtual-Machines-Should-Not-Be-Preview.test.ps1

```diff
@@ -0,0 +1,21 @@
+﻿param(
+[Parameter(Mandatory=$true)]
+[PSObject]
+$TemplateObject
+)
+foreach ($resource in $templateObject.resources) {
+    if ('microsoft.compute/virtualmachinescalesets', 'microsoft.compute/virtualmachines' -notcontains $resource.ResourceType) {

```

Comment:
can you document this or use -or operator since it will be a little advanced for some